src/core/manager-serialize.c

```diff
@@ -516,7 +523,26 @@ int manager_deserialize(Manager *m, FILE *f, FDSet *fds) {
 
                         if (strv_extend(&m->deserialized_subscribed, val) < 0)
                                 return -ENOMEM;
+                } else if ((val = startswith(l, "varlink-server-socket-address="))) {
+                        if (!m->varlink_server && MANAGER_IS_SYSTEM(m)) {
+                                r = manager_setup_varlink_server(m, &m->varlink_server);
+                                if (r < 0) {
+                                        log_warning_errno(r, "Failed to setup varlink server, ignoring: %m");
+                                        continue;
+                                }
+
+                                r = varlink_server_attach_event(m->varlink_server, m->event, SD_EVENT_PRIORITY_NORMAL);
+                                if (r < 0) {
+                                        log_warning_errno(r, "Failed to attach varlink connection to event loop, ignoring: %m");
+                                        m->varlink_server = varlink_server_unref(m->varlink_server);
+                                        continue;
+                                }
+
+                                deserialize_varlink_sockets = true;
```

Comment:
Is this flag necessary??

Reply:
I think so? It's only set if we have to create a new `m->varlink_server` during deserialization. This is to distinguish between cases when the manager deserialize code is called while `m->varlink_server` is still valid (such as during `systemctl daemon-reload`)

Reply:
You mean, here we should only the first address and fd are deserialized?? If so, then maybe better to log something when we ignore the line, and please add a brief comment about that.